pallets/manta-pay/src/test/payment.rs

```diff
@@ -616,3 +619,84 @@ fn pull_ledger_diff_should_work() {
         assert_eq!(runtime_pull_response.senders, decoded_senders);
     });
 }
+
+fn assert_manta_pay_suspension() {
+    assert_eq!(
+        TransactionPause::paused_transactions((b"MantaPay".to_vec(), b"to_private".to_vec())),
+        Some(())
+    );
+
+    assert_eq!(
+        TransactionPause::paused_transactions((b"MantaPay".to_vec(), b"private_transfer".to_vec())),
+        Some(())
+    );
+
+    assert_eq!(
+        TransactionPause::paused_transactions((b"MantaPay".to_vec(), b"to_public".to_vec())),
+        Some(())
+    );
+
+    assert_eq!(
+        TransactionPause::paused_transactions((b"MantaPay".to_vec(), b"public_transfer".to_vec())),
```

Comment:
I know it's test case, but once we really get this ledger error, `public_transfer` should not be shutdown,  imo.

Reply:
Actually you're probably right, pallet-assets transfer is disabled.
But we can just enable pallet-assets transfer instead.

Reply:
Ok , removed public transfer puase for now.